jetty9/src/main/docker/Dockerfile

```diff
@@ -14,35 +14,28 @@
 
 FROM ${docker.openjdk.image}
 
-# create jetty group and user
-RUN groupadd -r jetty \
- && useradd -r -g jetty jetty
-
 # Create env vars to identify image
 ENV JETTY_VERSION ${jetty9.version}
 ENV GAE_IMAGE_NAME jetty
 ENV GAE_IMAGE_LABEL ${docker.tag.long}
 
 # Create Jetty Home
-ENV JETTY_HOME /usr/local/jetty
+ENV JETTY_HOME ${jetty.home}
 ENV PATH $JETTY_HOME/bin:$PATH
 ENV TMPDIR /tmp/jetty
-ADD jetty-distribution.tar.gz /usr/local/
-RUN mv /usr/local/jetty-distribution-* $JETTY_HOME \
- && chown -R jetty:jetty $JETTY_HOME
+RUN mkdir $TMPDIR
+ADD jetty-distribution.tar.gz $JETTY_HOME/..
```

Comment:
I can't promise it is 100% drop in replacement right now, it should be close and if it isn't then likely something we need to tweak because it is ultimately meant to be used for these sorts of things.